test/parallel/test-buffer.js

```diff
@@ -1170,3 +1170,75 @@ assert.throws(function() {
 
 // Regression test for https://github.com/iojs/io.js/issues/649.
 assert.throws(function() { Buffer(1422561062959).toString('utf8'); });
+
+// Test indexOf
```

Comment:
would this be better in a separate test file, such as `test-buffer-indexof.js` to avoid test file bloat? `test-crypto` had the same issue until it was addressed in #827